spec/ruby/core/kernel/format_spec.rb

```diff
@@ -11,4 +11,20 @@
   it "is accessible as a module function" do
     Kernel.format("%s", "hello").should == "hello"
   end
+
+  it "formats string with precision" do
+    Kernel.format("%.3s", "hello").should == "hel"
+    Kernel.format("%-3.3s", "hello").should == "hel"
+  end
+
+  describe "on multibyte strings" do
+    it "formats string with precision" do
+      Kernel.format("%.3s", "hello".encode('UTF-16LE')).should == "hel"
```

Comment:
This spec fails on CRuby (see https://github.com/oracle/truffleruby/pull/2308/checks?check_run_id=2289342217), probably it needs to be `== "hel".encode('UTF-16LE')`